foundry/chatBubbles.d.ts

```diff
@@ -20,10 +30,53 @@ declare class ChatBubbles {
 
   /**
    * Speak a message as a particular Token, displaying it as a chat bubble
-   * @param token - The speaking Token
+   * @param token   - The speaking Token
    * @param message - The spoken message text
-   * @param emote - Whether to style the speech bubble as an emote
+   * @param emote   - Whether to style the speech bubble as an emote
    * @returns A Promise which resolves once the chat bubble has been created
    */
   say(token: Token, message: string, { emote }: { emote: boolean }): Promise<void>;
+
+  /**
+   * Clear any existing chat bubble for a certain Token
+   */
+  protected _clearBubble(token: Token): Promise<void>;
+
+  /**
+   * Render the HTML template for the chat bubble
+   * @param data - Template data
+   * @returns The rendered HTML
+   */
+  protected _renderHTML({ token, message, emote }: { token: Token; message: string; emote: boolean }): Promise<string>;
```

Comment:
destructuring of the parameters should be used if and only if it is done in foundry (to match the JSDoc). So in this case, just call the parameter `data`.